codegen/method.go

```diff
@@ -64,8 +62,7 @@ type MethodSpec struct {
 	ExceptionsIndex  map[string]ExceptionSpec
 	ValidStatusCodes []int
 	// Additional struct generated from the bundle of request args.
-	RequestBoxed  bool
-	RequestStruct []StructSpec
+	RequestBoxed bool
```

Comment:
I think we kind of support it. We want to get rid of it, but that's a different PR.